analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/HardcodedIpAddress.cs

```diff
@@ -55,8 +59,11 @@ public HardcodedIpAddress()
             string loopbackAddress1 = "127.0.0.1"; //Compliant
             string loopbackAddress2 = "127.2.3.4"; //Compliant
             string nonRoutableAddress = "0.0.0.0"; //Compliant
+            string documentationRange1 = "192.0.2.111"; //Compliant
+            string documentationRange2 = "198.51.100.111"; //Compliant
+            string documentationRange3 = "203.0.113.111"; //Compliant
```

Comment:
This would look way better if the `// Compliant`comments havethe same indent. Consider also aligning the other ones above and below.
Or maybe just removing the comment, because it doesn't explain why anyway, it's in the name